src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -13,7 +13,7 @@ function complexElements(nodes: NodeListOf<ChildNode>): ChildNode[] {
   return Array.from(nodes).filter((n) => n.nodeType !== 8);
 }
 
-type SupportedTagNames = 'event' | 'geogname' | 'note' | 'orgname' | 'persname' | 'placename' | 'ptr';
+type SupportedTagNames = 'event' | 'geogname' | 'note' | 'orgname' | 'persname' | 'placename' | 'ptr' | 'p';
```

Comment:
Shall we order alphabetically also this list?

Reply:
Maybe it's better, but can we do it in another PR?

Reply:
Maybe it's better!

Reply:
Yeah, let's do it!